mix.exs

```diff
@@ -92,9 +92,7 @@ defmodule Screens.MixProject do
       {:telemetry, "~> 1.2"},
       {:telemetry_poller, "~> 1.0"},
       {:telemetry_metrics, "~> 1.0"},
-      {:screens_config,
-       git: "https://github.com/mbta/screens-config-lib.git",
-       ref: "b2ffefe272d47a4e3173e3eea77cd0e2a84b8179"},
+      {:screens_config, github: "mbta/screens-config-lib"},
```

Comment:
I guess there'd never be a change to screens-config-lib that we wouldn't see review right?

Out of curiosity was there a reason we were holding onto a specific commit previously?

Reply:
oh I missed this last commit before approving! I liked pointing at a specific commit for tracking changes more accurately and in case of the need for a rollback.

Reply:
My reasoning here is that the lockfile does point to a specific commit, similarly to how e.g. we depend on `{:jason, "~> 1.0"}` and that resolves and is locked to a specific version. There's no way for the underlying version/commit hash to be updated without some explicit action on our part or without a change showing up in `mix.lock`. If we did go through that, merged the change, and it turned out we shouldn't have, the process of rolling it back or pinning it to an older version would be the same as for other dependencies.

Also, this should allow Dependabot to open PRs for updates to this library, to remind us if we've forgotten to bring it up to date.

Reply:
ah I didn't realize the lockfile would still point to a specific version. This update makes sense to me in that case!